### PR TITLE
Complete Navbar

### PR DESCRIPTION
**Pull request by joe-studer-18**
Closes issue #22. See issue for a screenshot of the new navbar (slightly different from final version).

### CODE REVIEW

**Comment by joe-studer-18** [comment]
Just bumping this for review @jaredwhanisch @noahklein00 . Ignore the unsuccessful checks. I ended up tossing those entirely for now. I'm having an issue getting them to run appropriately & am trying to figure it out in one of my private repos.

File: active/views/about.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <html>
 <base href="../" />
 
+<base href="../" />
+
```
Comment (jaredwhanisch) [inline active/views/about.html:6]: Is this necessary to have the base href declared twice?

File: active/index.html [hunk 2]
```diff
@@ -44,38 +37,29 @@
...
-                    <!--/.nav-collapse -->
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="#">
+                        <img src="img/KA_logo.png"></img>
```
Comment (jaredwhanisch) [inline active/index.html:48]: I don't think you included this image into the directory. It's not showing on the webpage when I run it, and it isn't included in the PR.

Comment (joe-studer-18) [reply active/index.html:48]: Nice catch, totally missed that.

File: active/views/about.html (hunk 1 above)
Comment (joe-studer-18) [reply active/views/about.html:6]: Hmm, Idk why i did this. Thanks for pointing it out.

File: active/index.html (hunk 2 above)
Comment (jaredwhanisch) [reply active/index.html:48]: Sorry to keep nit picking at this lol, but I believe you mis-named the file as KA_Logo.png instead of KA_logo.png.

**Comment by joe-studer-18** [comment]
Lol lets blame these mistakes on lack of sleep or something @jaredwhanisch. Should be g2g now.

**Pull request merged**